src/data/actions/instructorInfo.js

```diff
@@ -33,3 +36,36 @@ export function createInstructor(instructorData) {
       });
   };
 }
+
+export function requestInstructorInfoFail(error) {
+  return { type: REQUEST_INSTRUCTOR_INFO_FAIL, error };
+}
+
+export function requestInstructorInfoSuccess(data) {
+  return { type: REQUEST_INSTRUCTOR_INFO_SUCCESS, data };
+}
+
+export function requestInstructorInfo() {
+  return { type: REQUEST_INSTRUCTOR_INFO };
+}
+
+export function fetchInstructorInfo(id) {
```

Comment:
We get uuids for instructors rather than ids

Reply:
Fixed